website/docs/cluster-management/getting-started.mdx

```diff
@@ -175,6 +175,8 @@ Download the profile repository below to your config repository path then commit
   {ProfileRepo}
 </CodeBlock>
 
+For more information about profiles, see [profiles from private helm repositories](./profiles.mdx/#optional-using-a-helm-chart-from-a-remote-publicprivate-repository), [policy profiles](../policy/weave-policy-profile.mdx), and [eso secrets profiles](../secrets/setup-eso.mdx).
```

Comment:
given that the scope of the PR is to extend profiles documentation for the use case where the repo is private, i guess that you could remove 

>[policy profiles](../policy/weave-policy-profile.mdx), and [eso secrets profiles](../secrets/setup-eso.mdx).

as these are two examples of profiles ... or you want them for some other reason?